include/mbgl/renderer/renderer.hpp

```diff
@@ -21,6 +21,17 @@ namespace gfx {
 class RendererBackend;
 } // namespace gfx
 
+struct PlacedSymbolData {
+    std::u16string key;
+    std::size_t dataFeatureIndex;
```

Comment:
dataFeatureIndex doesn't bring much of a value, shouldn't it be feature id, right?

Reply:
This is taken after `SymbolInstance` declaration
`std::size_t dataFeatureIndex;   // Index into the underlying tile data feature set`

Perhaps it can be renamed, but needs to be changed everywhere in the code base for consistency

Reply:
Removed the field as it's not quite useful